.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint-staged
```

Comment:
shouldn't this be exposed in our root level package.json as a new script entry, i.e. `yarn run lint-staged`? Or am I missing something about this workflow...

Reply:
See: https://github.com/okta/odyssey/pull/1065/files#r714947632